.circleci/config.yml

```diff
@@ -1,80 +1,85 @@
 version: 2.1
 
-default_workflow_jobs: &default_workflow_jobs
+.push-context: &push-context
+  context: dockerhub
+  filters:
+    branches:
+      only: 
+        - master
+  requires:
+    - test-cli
+    - test-fpm
+    - test-http
+    - scan-vulnerability
+
+.scheduled_workflow_jobs: &scheduled_workflow_jobs
   - lint
   - lint-shell
-  - build-http:
+  - build-http: &build-http
```

Comment:
The next three are all the same and then there's three that are again all the same. Why do we not have one of each and also one alias?

Reply:
Yeah I want to make it looking better, but I'm suffering with this yaml and circle ci. Will try more later

Reply:
![image](https://user-images.githubusercontent.com/70739/54764529-48fca400-4bf8-11e9-8906-dd863594c9a9.png)